tests/begin_test.go

```diff
@@ -25,16 +28,129 @@ import (
 	"github.com/kylelemons/godebug/pretty"
 )
 
-func TestBegin(t *testing.T) {
-	// Call begin
+func TestBeginEmpty(t *testing.T) {
 	tmpdir, err := setUpTest()
 	if err != nil {
 		t.Fatalf("%v", err)
 	}
 	defer cleanUpTest(tmpdir)
 
+	expectedMan := schema.ImageManifest{
```

Comment:
style nit, "wim" ("wanted image manifest") would be a more idiomatic name. test convention is ~usually to use g(ot), w(ant)